tools/run_tests/build_artifact_python.sh

```diff
@@ -46,7 +46,7 @@ fi
 GRPC_PYTHON_USE_CUSTOM_BDIST=0  \
 GRPC_PYTHON_BUILD_WITH_CYTHON=1 \
 ${SETARCH_CMD} python setup.py  \
-    bdist_wheel_grpc_custom
+    build_tagged_ext
 
 GRPC_PYTHON_USE_CUSTOM_BDIST=0  \
 GRPC_PYTHON_BUILD_WITH_CYTHON=1 \
```

Comment:
bdist_wheel should only be built on mac, but for now it's fine (let's fix this in a followup PR)